frontend/host-app/src/components/CreateEventModal/InputHashTag.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import {styled} from "@material-ui/core/styles";
+import {TextField} from "@material-ui/core";
+import uuidv1 from "uuid/v1";
+
+const ENTER_KEY = 13;
+const CustomTextField = styled(TextField)({
+	width: 400,
+});
+
+function InputHashTag(props) {
+	const prevHashTagList = props.hashTags;
+	const addHashTag = event => {
+		if (event.keyCode === ENTER_KEY) {
+			if (event.target.value.length > 28) {
```

Comment:
```suggestion
			if (event.target.value.length > SOME_GREAT_REASON) {
```
28은 너무 모호합니다. 좀더 의미있는 변수이면 좋을것 같습니다.